app/services/hydranorth/ezid_service.rb

```diff
@@ -0,0 +1,91 @@
+module Hydranorth
+  class EzidService
+
+    def initialize()
```

Comment:
It doesn't seem like there needs to be instances of the EzidService. All of it's methods could be class methods rather than instance methods -- kind of a singleton design pattern. This would be a subtle optimization because I think '.new' will create a new instance in memory each time it is called. Because there is no state saved by an instance of the EzidService this isn't necessary. Might be worth bringing this topic to a BoF.